### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by ArneTR** [Green-Software-Foundation/awesome-green-software#169] [General]
PRs are dating back over a year now.

Happy to support if you need volunteers.

**Comment by prof-antar** [comment]
Hi @ArneTR, we just had new project assignments this month. Good news, the project is being revamped and will have an enhanced process for contribution over the next month. Stay tuned for more updates.

**Comment by ArneTR** [comment]
Great to hear, thanks for the update!

**Comment by mueller-fr** [comment]
As of now, PRs are dating back a month only. Looks good.
